roles/backend_setup/tasks/bind_tang_server.yml

```diff
@@ -25,21 +25,23 @@
     index_var: index
   when: gluster_infra_tangservers is defined
 
+# TODO Need to investigate how to configure incase of static deployment
 - name: Add network entry on /etc/dracut.conf.d/clevis.conf
   blockinfile:
     path: /etc/dracut.conf.d/clevis.conf
     create: yes
     state: present
     block: |
       kernel_cmdline="ip={{ networkinterface }}:dhcp"
-      omit_dracutmodules+="{{ networkinterface }}"
+      omit_dracutmodules+="ifcfg"
     marker: "# {mark} Entry for {{networkinterface}}"
   when: gluster_infra_tangservers is defined
 
-# Once "dracut -vf" execution is not working, so executing two times till finding issue. 
 - name: Execute dracut -vf to configure tang
-  command: dracut -vf
-  with_items:
-    - 1
-    - 2
+  command: dracut -vf --regenerate-all
```

Comment:
wouldnt dracut -vf require sudo permissions to write to boot ?

Reply:
The playbook will be running with root user, so sudo does not requird.